### ISSUE TITLE
Standardized codes for prescription and diagnosis

### ISSUE THREAD

**Issue by topspinj** [MIT-LCP/mimic-code#499]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.physionet.org/about/mimic/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

I have a few questions about MIMIC's data collection methodology:

1. Diagnosis: Were diagnoses coded with ICD-9 at data entry, or was this a post-processing step done by the MIMIC team?
2. Prescription names: How is prescription information entered into a patient's medical record? Is this done via clinical notes, or a separate order form? It seems like `drug` in the prescriptions table is free-form text. How were `drug_name_poe` and `drug_name_generic` generated? Was this part of the data entry process performed by the caregiver? 
3. Age denoted in clinical notes: Just want to confirm that age was intentionally de-identified in clinical notes. I have found some cases of reported age varying drastically in different clinical notes for the same patient admission.

Thank you!

**Comment by alistairewj** [comment]
1. https://mimic.physionet.org/mimictables/diagnoses_icd/
> The ICD codes are generated for billing purposes at the end of the hospital stay.

2. POE is provider order entry, which is a short hand for the drug used in the back end database. Certainly an order is made by the caregiver, but I'm not sure how the various names for the drugs show up. More than likely they enter `drug` and `drug_name_generic` is automatically assigned.

3. Yes, ages > 89 were deidentified in the notes.

**Comment by topspinj** [comment]
Re. 3: I've noticed that for patients under 89, their age also varies across clinical notes. For example, in one particular hospital admission, a patient's age fluctuated between 22 and 49 years old. I calculated the patient's age by subtracting `dob` from `admitdate`, and it turns out the patient was 49. This occurs in other situations, and I wanted to confirm that these notes belong to the same patient despite having different reported ages.
